app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -282,9 +282,10 @@ class WireActivity : AppCompatActivity() {
             val onComplete: (convId: ConversationId?) -> Unit = remember {
                 {
                     it?.also {
-                        appLogger.d("Join conversation via code dialog completed, navigating to conversation screen")
                         navigate(NavigationCommand(ConversationScreenDestination(it), BackStackMode.CLEAR_TILL_START))
                         viewModel.onJoinConversationFlowCompleted()
+                    } ?: run {
+                        viewModel.onJoinConversationFlowCompleted()
```

Comment:
what about calling it `onJoinConversationFlowCompleted` once after `it.also{}` ?